package.json

```diff
@@ -41,13 +42,15 @@
     "Frank Natividad <frankyn@users.noreply.github.com>",
     "Jason Dobry <jason.dobry@gmail.com>",
     "Jason Dobry <jmdobry@users.noreply.github.com>",
+    "Justin Beckwith <justin.beckwith@gmail.com>",
     "Justin Sprigg <justin.sprigg@gmail.com>",
     "Luke Sneeringer <luke@sneeringer.com>",
     "Stephen <stephenplusplus@users.noreply.github.com>",
     "Stephen Sawchuk <sawchuk@gmail.com>",
     "Stephen Sawchuk <stephenplusplus@users.noreply.github.com>",
     "Tyler Johnson <mail@tyler-johnson.ca>",
     "Zach Bjornson <bjornson@stanford.edu>",
+    "Zach Bjornson <zbbjornson@gmail.com>",
```

Comment:
so this is not related to your change anyway, but since you are regenerating stuff, can you please add Zach to `.mailmap` to make sure he's listed just once?

Reply:
Done